### ISSUE TITLE
[1.2.0 Beta] Myriad quest completion issues

### ISSUE THREAD

**Issue by L-Clouds** [rgtexa/Automaton#14]
tl;dr:
Burninate isn't detecting that I'm crafting a furnace on a vanilla crafting table.  Maybe it's because I crafted a furnace before finishing the prerequisite Resource Collection; I haven't gone back and tested that specifically.  (In a different world, it detected without problem.)

Nosy Neighbors isn't detecting most mob kills.  I had to kill way more than 5 zombies to get 5/5 zombies; I've slain numerous skeletons and creepers and am still at 0/5 for both of those.  (I only even knew about this because I saw the quest when edit mode was active - the quest is marked to be invisible until complete???)

Welcome to Chisel Town detected when I crafted the Chisel, but not when I crafted the Iron Chisel.

--------------------------------------------------------------------------------

I started playing with a local Forge server on a world I'd generated in creative mode, with cheats, in single player and then moved to the server.  I'd also added some extra quality-of-life mods (AppleSkin, FindMe, JustEnoughButtons).  It wasn't until a little bit later that I realized that BQ edit mode was on (due to generating with cheats enabled) so I turned it off.

In this world, I had the problems with Nosy Neighbors and Welcome to Chisel Town.  Resetting specific quests with /bq_admin didn't resolve it; resetting my whole player quest progression didn't either.

Thinking that maybe it's due to any of those things, I downloaded a clean instance of Automaton 1.2.0 and generated a non-cheats, survival, single-player world with unaltered mods and configs, and tried to rush to those quests.

That's when I had a problem with Burninate, and gave up.

I have the logs still but I don't see anything relevant in them.

**Comment by rgtexa** [comment]
The beta was using a beta version of Better Questing that was in development of a complete re-write. It's likely that this is what caused the issues you saw.

Automaton 1.3.0 was released today with an updated stable version of Better Questing and I've not experienced any issues with the quests detecting or completing properly in testing with this version.

As a note for Crafting requirement quests - they do require the crafting of the item when the quest is available. They generally don't detect and complete the task if the quest is still "locked."

Recommend updating the pack to this version.
